yapf/VERSION

```diff
@@ -0,0 +1 @@
+0.40.2
```

Comment:
HACKING.md should be updated to describe where to bump the version number -- currently it suggests pyproject.toml